Formula/portal.rb

```diff
@@ -0,0 +1,61 @@
+class Portal < Formula
+  desc "Quick and easy command-line file transfer utility from any computer to another"
+  homepage "https://github.com/SpatiumPortae/portal"
+  url "https://github.com/SpatiumPortae/portal/archive/v1.2.2.tar.gz"
+  sha256 "94c334285925175edfc470239ea2cc73a0039f9487ce92e60764f0965aba66d4"
+  license "MIT"
+  head "https://github.com/SpatiumPortae/portal.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Is this needed? Does `brew livecheck portal` return the wrong output without it?

Reply:
It was not needed, the default strategy seems to work.